client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/Sections/Details.tsx

```diff
@@ -0,0 +1,290 @@
+                  value={t('format.quantity-with-months', {
+                    count: minMonthsOfStock ?? maxMonthsOfStock,
+                  })}
+                  style={{ textAlign: 'right' }}
+                />
+              }
+              labelWidth={LABEL_WIDTH}
+              label={t('label.reorder-threshold')}
+            />
+            <InputWithLabelRow
+              Input={
+                <ReadOnlyInput
+                  value={`${formatNumber.round(targetStock)} ${t('label.units-plural', { count: targetStock })}`}
+                  style={{ textAlign: 'right' }}
+                />
+              }
+              labelWidth={LABEL_WIDTH}
+              label={t('label.target-stock')}
+            />
+            <InputWithLabelRow
+              Input={
+                <ReadOnlyInput
+                  value={t('format.quantity-with-units', {
+                    count: NumUtils.round(draft?.suggestedQuantity ?? 0),
+                  })}
+                  style={{ textAlign: 'right' }}
+                />
+              }
+              labelWidth={LABEL_WIDTH}
+              label={t('label.suggested-quantity')}
+            />
+            {isProgram && useConsumptionData && (
+              <>
+                <InputWithLabelRow
+                  Input={
+                    <NumericReadOnlyInput
+                      value={draft?.itemStats.availableMonthsOfStockOnHand}
+                    />
+                  }
+                  labelWidth={LABEL_WIDTH}
+                  label={t('label.months-of-stock')}
+                />
+                <InputWithLabelRow
+                  Input={
+                    <ReadOnlyInput
+                      value={t('format.quantity-with-units', {
+                        count: draft?.expiringUnits,
+                      })}
+                      style={{ textAlign: 'right' }}
+                    />
+                  }
+                  labelWidth={LABEL_WIDTH}
+                  label={t('label.short-expiry')}
+                />
+                <InputWithLabelRow
+                  Input={<NumericReadOnlyInput value={draft?.daysOutOfStock} />}
+                  labelWidth={LABEL_WIDTH}
+                  label={t('label.days-out-of-stock')}
+                />
+                <InputWithLabelRow
+                  Input={
+                    <ReasonOptionsSearchInput
+                      value={draft?.reason}
+                      onChange={value => {
+                        update({ reason: value });
+                      }}
+                      width={150}
+                      type={ReasonOptionNodeType.RequisitionLineVariance}
+                      isDisabled={
+                        draft?.requestedQuantity === draft?.suggestedQuantity ||
+                        disabled
+                      }
+                      onBlur={save}
+                    />
+                  }
+                  labelWidth={'66px'}
```

Comment:
rip feel like its only a matter of time before we add a translation wider than this 👀